var/spack/repos/builtin/packages/python-dev/package.py

```diff
@@ -11,9 +11,9 @@ class PythonDev(Package):
     """Meta package to bundle python packages for development"""
 
     homepage = "http://www.dummy.org/"
-    url      = "https://www.dummy.org/source/dummy-0.1.zip"
+    url      = "https://www.dummy.org/source/dummy-0.2.zip"
```

Comment:
For future reference, this version change here is not really needed. Spack will be able to guess the version.

Reply:
I guess, but just to keep it consistent :)